### DISCUSSION TITLE
Multiple Context

### DISCUSSION THREAD

**Discussion by ricaragao** [Natizyskunk/vscode-sftp#434] [Q&A]
Hi,

I configured two different contexts, but just the last one is working, but if I use each one individually, they work fine. 

And in the menu to upload, it just show me UPLOAD, not options to choose the context.

thanks

**Comment by Natizyskunk** [comment]
@ricaragao,

Could you be a little bit mor explicit on what exactly is your problem ?
Can you attach some screenshots ?
